docs/source/2x_user_guide.md

```diff
@@ -67,7 +67,7 @@ This part provides the advanced topics that help user dive deep into Intel® Neu
 <td colspan="6" align="center"><a href="add_new_adaptor.md">Add New Adaptor</a></td>
 </tr>
 <tr>
-<td colspan="3" align="center"><a href="distillation_quantization.md">Distillation for Quantization</a></td>
+<td colspan="3" align="center"><a href="distillation.md">Distillation</a></td>
```

Comment:
Will you add this doc back?

Reply:
This file was added by https://github.com/intel/neural-compressor/pull/2205/files.